src/Cake.Core/IO/ProcessRunner.cs

```diff
@@ -48,13 +48,16 @@ public IProcess Start(FilePath filePath, ProcessSettings settings)
                 throw new ArgumentNullException("settings");
             }
 
+            // Get the fileName
+            var fileName = _environment.IsUnix() ? filePath.FullPath : filePath.FullPath.Quote();
```

Comment:
This variable name confused me a little since it's the whole path that this contains and not just the file name. Maybe we can change this to `fullFileName` or something that shows that this is the full path to the file?